translate/storage/mozilla_lang.py

```diff
@@ -116,6 +130,13 @@ def parse(self, lines):
     def serialize(self, out):
         if self.is_active or self.mark_active:
             out.write(b"## active ##\n")
+        for header in self._headers:
```

Comment:
I would use `line` and `self.header`. It is not like you have multiple headers at the beginning of the file, just 1 header with multiple lines.